mlir/include/mlir/Dialect/ArmSME/IR/CMakeLists.txt

```diff
@@ -1,12 +1,24 @@
 add_mlir_dialect(ArmSME arm_sme ArmSME)
 add_mlir_doc(ArmSME ArmSME Dialects/ -gen-dialect-doc -dialect=arm_sme)
```

Comment:
Note that with these changes the docs will be empty as `ArmSME.td` does not include the op definitions. This was the same problem the vector dialect had a little while ago.

Reply:
#68110 ;-)

Reply:
No worries then :smile: